tests/Integration/Behaviour/Features/Context/ProductFeatureContext.php

```diff
@@ -83,13 +85,39 @@ public function before(BeforeScenarioScope $scope)
     /* PRODUCTS */
 
     /**
-     * @param $productName
+     * @param string $productName
      *
      * @return Product
      */
-    public function getProductWithName($productName)
+    public function getProduct(string $productName)
```

Comment:
Was it really necessary to rename this one? I'm just anticipating the potential conflicts and missing functions that may come after this one is merged, and the change of name doesn't seem to bring much value except reduce the code size in this class

Reply:
Fixed